### ISSUE TITLE
'NoneType' object has no attribute 'video' error. Full terminal log posted

### ISSUE THREAD

**Issue by evanwins** [mdhiggins/sickbeard_mp4_automator#1116]
I've been getting this error since I updated md5 and I cannot figure out what it is.  I don't know quite what I did but the script isn't working either manually or automatically as part of Sabs/Radarr.

Here is the full log of what happened in Terminal:

MiniMCHD:sickbeard_mp4_automator MiniMCHD$ python manual.py --auto
ERROR:root:code for hash md5 was not found.
Traceback (most recent call last):
  File "/usr/local/Cellar/python@2/2.7.14_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/hashlib.py", line 147, in <module>
    globals()[__func_name] = __get_hash(__func_name)
  File "/usr/local/Cellar/python@2/2.7.14_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/hashlib.py", line 97, in __get_builtin_constructor
    raise ValueError('unsupported hash type ' + name)
ValueError: unsupported hash type md5
ERROR:root:code for hash sha1 was not found.
Traceback (most recent call last):
  File "/usr/local/Cellar/python@2/2.7.14_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/hashlib.py", line 147, in <module>
    globals()[__func_name] = __get_hash(__func_name)
  File "/usr/local/Cellar/python@2/2.7.14_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/hashlib.py", line 97, in __get_builtin_constructor
    raise ValueError('unsupported hash type ' + name)
ValueError: unsupported hash type sha1
ERROR:root:code for hash sha224 was not found.
Traceback (most recent call last):
  File "/usr/local/Cellar/python@2/2.7.14_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/hashlib.py", line 147, in <module>
    globals()[__func_name] = __get_hash(__func_name)
  File "/usr/local/Cellar/python@2/2.7.14_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/hashlib.py", line 97, in __get_builtin_constructor
    raise ValueError('unsupported hash type ' + name)
ValueError: unsupported hash type sha224
ERROR:root:code for hash sha256 was not found.
Traceback (most recent call last):
  File "/usr/local/Cellar/python@2/2.7.14_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/hashlib.py", line 147, in <module>
    globals()[__func_name] = __get_hash(__func_name)
  File "/usr/local/Cellar/python@2/2.7.14_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/hashlib.py", line 97, in __get_builtin_constructor
    raise ValueError('unsupported hash type ' + name)
ValueError: unsupported hash type sha256
ERROR:root:code for hash sha384 was not found.
Traceback (most recent call last):
  File "/usr/local/Cellar/python@2/2.7.14_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/hashlib.py", line 147, in <module>
    globals()[__func_name] = __get_hash(__func_name)
  File "/usr/local/Cellar/python@2/2.7.14_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/hashlib.py", line 97, in __get_builtin_constructor
    raise ValueError('unsupported hash type ' + name)
ValueError: unsupported hash type sha384
ERROR:root:code for hash sha512 was not found.
Traceback (most recent call last):
  File "/usr/local/Cellar/python@2/2.7.14_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/hashlib.py", line 147, in <module>
    globals()[__func_name] = __get_hash(__func_name)
  File "/usr/local/Cellar/python@2/2.7.14_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/hashlib.py", line 97, in __get_builtin_constructor
    raise ValueError('unsupported hash type ' + name)
ValueError: unsupported hash type sha512
Traceback (most recent call last):
  File "manual.py", line 12, in <module>
    from tvdb_mp4 import Tvdb_mp4
  File "/sickbeard_mp4_automator/tvdb_mp4.py", line 15, in <module>
    from tvdb_api.tvdb_api import Tvdb
  File "/sickbeard_mp4_automator/tvdb_api/tvdb_api.py", line 29, in <module>
    from tvdb_cache import CacheHandler
  File "/sickbeard_mp4_automator/tvdb_api/tvdb_cache.py", line 23, in <module>
    from hashlib import md5
ImportError: cannot import name md5
MiniMCHD:sickbeard_mp4_automator MiniMCHD$ python manual.py --auto
ERROR:root:code for hash md5 was not found.
Traceback (most recent call last):
  File "/usr/local/Cellar/python@2/2.7.14_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/hashlib.py", line 147, in <module>
    globals()[__func_name] = __get_hash(__func_name)
  File "/usr/local/Cellar/python@2/2.7.14_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/hashlib.py", line 97, in __get_builtin_constructor
    raise ValueError('unsupported hash type ' + name)
ValueError: unsupported hash type md5
ERROR:root:code for hash sha1 was not found.
Traceback (most recent call last):
  File "/usr/local/Cellar/python@2/2.7.14_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/hashlib.py", line 147, in <module>
    globals()[__func_name] = __get_hash(__func_name)
  File "/usr/local/Cellar/python@2/2.7.14_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/hashlib.py", line 97, in __get_builtin_constructor
    raise ValueError('unsupported hash type ' + name)
ValueError: unsupported hash type sha1
ERROR:root:code for hash sha224 was not found.
Traceback (most recent call last):
  File "/usr/local/Cellar/python@2/2.7.14_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/hashlib.py", line 147, in <module>
    globals()[__func_name] = __get_hash(__func_name)
  File "/usr/local/Cellar/python@2/2.7.14_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/hashlib.py", line 97, in __get_builtin_constructor
    raise ValueError('unsupported hash type ' + name)
ValueError: unsupported hash type sha224
ERROR:root:code for hash sha256 was not found.
Traceback (most recent call last):
  File "/usr/local/Cellar/python@2/2.7.14_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/hashlib.py", line 147, in <module>
    globals()[__func_name] = __get_hash(__func_name)
  File "/usr/local/Cellar/python@2/2.7.14_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/hashlib.py", line 97, in __get_builtin_constructor
    raise ValueError('unsupported hash type ' + name)
ValueError: unsupported hash type sha256
ERROR:root:code for hash sha384 was not found.
Traceback (most recent call last):
  File "/usr/local/Cellar/python@2/2.7.14_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/hashlib.py", line 147, in <module>
    globals()[__func_name] = __get_hash(__func_name)
  File "/usr/local/Cellar/python@2/2.7.14_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/hashlib.py", line 97, in __get_builtin_constructor
    raise ValueError('unsupported hash type ' + name)
ValueError: unsupported hash type sha384
ERROR:root:code for hash sha512 was not found.
Traceback (most recent call last):
  File "/usr/local/Cellar/python@2/2.7.14_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/hashlib.py", line 147, in <module>
    globals()[__func_name] = __get_hash(__func_name)
  File "/usr/local/Cellar/python@2/2.7.14_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/hashlib.py", line 97, in __get_builtin_constructor
    raise ValueError('unsupported hash type ' + name)
ValueError: unsupported hash type sha512
Traceback (most recent call last):
  File "manual.py", line 12, in <module>
    from tvdb_mp4 import Tvdb_mp4
  File "/sickbeard_mp4_automator/tvdb_mp4.py", line 15, in <module>
    from tvdb_api.tvdb_api import Tvdb
  File "/sickbeard_mp4_automator/tvdb_api/tvdb_api.py", line 29, in <module>
    from tvdb_cache import CacheHandler
  File "/sickbeard_mp4_automator/tvdb_api/tvdb_cache.py", line 23, in <module>
    from hashlib import md5
ImportError: cannot import name md5
MiniMCHD:sickbeard_mp4_automator MiniMCHD$ brew reinstall python@2
Warning: You are using macOS 10.12.
We (and Apple) do not provide support for this old version.
You will encounter build failures with some formulae.
Please create pull requests instead of asking for help on Homebrew's GitHub,
Discourse, Twitter or IRC. You are responsible for resolving any issues you
experience while you are running this old version.

==> Reinstalling python@2 
==> Installing dependencies for python@2: gdbm, readline and sqlite
==> Installing python@2 dependency: gdbm
==> Downloading https://homebrew.bintray.com/bottles/gdbm-1.18.1.sierra.bottle.1
######################################################################## 100.0%
==> Pouring gdbm-1.18.1.sierra.bottle.1.tar.gz
🍺  /usr/local/Cellar/gdbm/1.18.1: 20 files, 586.8KB
==> Installing python@2 dependency: readline
==> Downloading https://homebrew.bintray.com/bottles/readline-8.0.1.sierra.bottl
==> Downloading from https://akamai.bintray.com/f2/f234d1ff8148bf08b0ac31e661f2e
######################################################################## 100.0%
==> Pouring readline-8.0.1.sierra.bottle.tar.gz
==> Caveats
readline is keg-only, which means it was not symlinked into /usr/local,
because macOS provides the BSD libedit library, which shadows libreadline.
In order to prevent conflicts when programs look for libreadline we are
defaulting this GNU Readline installation to keg-only.

For compilers to find readline you may need to set:
  export LDFLAGS="-L/usr/local/opt/readline/lib"
  export CPPFLAGS="-I/usr/local/opt/readline/include"

For pkg-config to find readline you may need to set:
  export PKG_CONFIG_PATH="/usr/local/opt/readline/lib/pkgconfig"

==> Summary
🍺  /usr/local/Cellar/readline/8.0.1: 48 files, 1.5MB
==> Installing python@2 dependency: sqlite
==> Downloading https://sqlite.org/2019/sqlite-autoconf-3300100.tar.gz
######################################################################## 100.0%
==> ./configure --prefix=/usr/local/Cellar/sqlite/3.30.1 --enable-dynamic-extens
==> make install
==> Caveats
sqlite is keg-only, which means it was not symlinked into /usr/local,
because macOS provides an older sqlite3.

If you need to have sqlite first in your PATH run:
  echo 'export PATH="/usr/local/opt/sqlite/bin:$PATH"' >> ~/.bash_profile

For compilers to find sqlite you may need to set:
  export LDFLAGS="-L/usr/local/opt/sqlite/lib"
  export CPPFLAGS="-I/usr/local/opt/sqlite/include"

For pkg-config to find sqlite you may need to set:
  export PKG_CONFIG_PATH="/usr/local/opt/sqlite/lib/pkgconfig"

==> Summary
🍺  /usr/local/Cellar/sqlite/3.30.1: 11 files, 3.9MB, built in 1 minute 40 seconds
==> Installing python@2
==> Downloading https://www.python.org/ftp/python/2.7.17/Python-2.7.17.tar.xz
######################################################################## 100.0%
==> ./configure --prefix=/usr/local/Cellar/python@2/2.7.17_1 --enable-ipv6 --dat
==> make
==> make install PYTHONAPPSDIR=/usr/local/Cellar/python@2/2.7.17_1
==> make frameworkinstallextras PYTHONAPPSDIR=/usr/local/Cellar/python@2/2.7.17_
==> Downloading https://files.pythonhosted.org/packages/f4/d5/a6c19dcbcbc267aca3
######################################################################## 100.0%
==> Downloading https://files.pythonhosted.org/packages/ce/ea/9b445176a65ae4ba22
######################################################################## 100.0%
==> Downloading https://files.pythonhosted.org/packages/59/b0/11710a598e1e148fb7
######################################################################## 100.0%
==> /usr/local/Cellar/python@2/2.7.17_1/bin/python -s setup.py --no-user-cfg ins
==> /usr/local/Cellar/python@2/2.7.17_1/bin/python -s setup.py --no-user-cfg ins
==> /usr/local/Cellar/python@2/2.7.17_1/bin/python -s setup.py --no-user-cfg ins
==> Caveats
Pip and setuptools have been installed. To update them
  pip install --upgrade pip setuptools

You can install Python packages with
  pip install <package>

They will install into the site-package directory
  /usr/local/lib/python2.7/site-packages

See: https://docs.brew.sh/Homebrew-and-Python
==> Summary
🍺  /usr/local/Cellar/python@2/2.7.17_1: 6,645 files, 91.2MB, built in 3 minutes 45 seconds
==> Caveats
==> readline
readline is keg-only, which means it was not symlinked into /usr/local,
because macOS provides the BSD libedit library, which shadows libreadline.
In order to prevent conflicts when programs look for libreadline we are
defaulting this GNU Readline installation to keg-only.

For compilers to find readline you may need to set:
  export LDFLAGS="-L/usr/local/opt/readline/lib"
  export CPPFLAGS="-I/usr/local/opt/readline/include"

For pkg-config to find readline you may need to set:
  export PKG_CONFIG_PATH="/usr/local/opt/readline/lib/pkgconfig"

==> sqlite
sqlite is keg-only, which means it was not symlinked into /usr/local,
because macOS provides an older sqlite3.

If you need to have sqlite first in your PATH run:
  echo 'export PATH="/usr/local/opt/sqlite/bin:$PATH"' >> ~/.bash_profile

For compilers to find sqlite you may need to set:
  export LDFLAGS="-L/usr/local/opt/sqlite/lib"
  export CPPFLAGS="-I/usr/local/opt/sqlite/include"

For pkg-config to find sqlite you may need to set:
  export PKG_CONFIG_PATH="/usr/local/opt/sqlite/lib/pkgconfig"

==> python@2
Pip and setuptools have been installed. To update them
  pip install --upgrade pip setuptools

You can install Python packages with
  pip install <package>

They will install into the site-package directory
  /usr/local/lib/python2.7/site-packages

See: https://docs.brew.sh/Homebrew-and-Python
MiniMCHD:sickbeard_mp4_automator MiniMCHD$ python manual.py --auto
Manual processor started.
/usr/local/Cellar/python@2/2.7.17_1/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python
64bit Python.
Enter path to file:
#: ^Ccd /Volumes/TeraPlex/AUTOMATE ;Traceback (most recent call last):
  File "manual.py", line 358, in <module>

    main()
  File "manual.py", line 325, in main
    path = getValue("Enter path to file")
  File "manual.py", line 53, in getValue
    value = raw_input("#: ").strip(' \"')
KeyboardInterrupt
MiniMCHD:sickbeard_mp4_automator MiniMCHD$ cd /Volumes/TeraPlex/AUTOMATE ;
MiniMCHD:AUTOMATE MiniMCHD$ cd /
MiniMCHD:/ MiniMCHD$ ls
1sickbeard_mp4_automator		home
Applications				installer.failurerequests
Library					net
Network					ombi
Ombi.git				private
System					requests-2.10.0-py2.py3-none-any.whl
Users					sbin
Volumes					sickbeard_mp4_automator
bin					tmp
cores					usr
dev					var
etc
MiniMCHD:/ MiniMCHD$ cd sickbeard_mp4_automator
MiniMCHD:sickbeard_mp4_automator MiniMCHD$ python manual.py --auto
Manual processor started.
/usr/local/Cellar/python@2/2.7.17_1/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python
64bit Python.
Enter path to file:
#:  /Volumes/TeraPlex/AUTOMATE
Processing file /Volumes/TeraPlex/AUTOMATE/Frozen.2.2019.DVDScr.XVID.AC3.HQ.Hive-CM8[oscars]/Frozen.2.2019.DVDScr.XVID.AC3.HQ.Hive-CM8.avi
Reading video stream.
An unexpected error occurred, processing of this file has failed
'NoneType' object has no attribute 'video'
MiniMCHD:sickbeard_mp4_automator MiniMCHD$ python manual.py --auto
Manual processor started.
/usr/local/Cellar/python@2/2.7.17_1/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python
64bit Python.
Enter path to file:
#: /Volumes/TeraPlex/AUTOMATE/DONE 
Processing file /Volumes/TeraPlex/AUTOMATE/DONE/The.Hustle.2019.1080p.BluRay.H264.AAC-RARBG/The.Hustle.2019.1080p.BluRay.H264.AAC-RARBG.mp4
Matched movie title as: The Hustle 2019-05-09
Processing The Hustle
An unexpected error occurred, processing of this file has failed
'NoneType' object has no attribute 'video'
MiniMCHD:sickbeard_mp4_automator MiniMCHD$ python manual.py --auto
Manual processor started.
/usr/local/Cellar/python@2/2.7.17_1/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python
64bit Python.
Enter path to file:
#: /Volumes/TeraPlex/AUTOMATE/DONE 
Processing file /Volumes/TeraPlex/AUTOMATE/DONE/The.Hustle.2019.1080p.BluRay.H264.AAC-RARBG/The.Hustle.2019.1080p.BluRay.H264.AAC-RARBG.mp4
Matched movie title as: The Hustle 2019-05-09
Processing The Hustle
An unexpected error occurred, processing of this file has failed
'NoneType' object has no attribute 'video'
MiniMCHD:sickbeard_mp4_automator MiniMCHD$ python manual.py --auto
Manual processor started.
/usr/local/Cellar/python@2/2.7.17_1/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python
64bit Python.
Enter path to file:
#: /Volumes/TeraPlex/AUTOMATE/DONE
Processing file /Volumes/TeraPlex/AUTOMATE/DONE/1917.2019.DVDScr.XVID.AC3.HQ.Hive-CM8[oscars]/1917.2019.DVDScr.XVID.AC3.HQ.Hive-CM8.avi
Reading video stream.
An unexpected error occurred, processing of this file has failed
'NoneType' object has no attribute 'video'
Processing file /Volumes/TeraPlex/AUTOMATE/DONE/Color.Out.of.Space.2019.1080p.SCREENER.x264-TOPKEK/Color.Out.of.Space.2019.1080p.SCREENER.x264-TOPKEK.mp4
Matched movie title as: Color Out of Space 2019-09-20
Processing Color Out of Space
An unexpected error occurred, processing of this file has failed
'NoneType' object has no attribute 'video'
Processing file /Volumes/TeraPlex/AUTOMATE/DONE/Dark.Waters.2019.1080p.SCREENER.x264.DTS-NOGRP/Dark.Waters.2019.1080p.SCREENER.x264.DTS-NOGRP.mkv
Matched movie title as: Dark Waters 2019-11-22
Processing Dark Waters
Reading video stream.
An unexpected error occurred, processing of this file has failed
'NoneType' object has no attribute 'video'
MiniMCHD:sickbeard_mp4_automator MiniMCHD$ which python
/usr/local/bin/python
MiniMCHD:sickbeard_mp4_automator MiniMCHD$ brew install Python3
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
==> New Formulae
ansible@2.8         dsvpn               libb64              wasmer
awscurl             findomain           mlpack              yaegi
bandwhich           git-filter-repo     mmctl
cgns                glow                nauty
cobalt              komposition         poetry
==> Updated Formulae
ffmpeg ✔                                 mafft
git ✔                                    magic-wormhole
jpeg ✔                                   mailutils
python ✔                                 makedepend
xvid ✔                                   man-db
abcmidi                                  mariadb
ack                                      mariadb@10.2
acpica                                   mariadb@10.3
algernon                                 mat2
aliyun-cli                               mawk
alp                                      maxwell
alpine                                   mbedtls
ammonite-repl                            mda-lv2
angular-cli                              mdcat
ansible                                  mdds
ansible-lint                             mercurial
apache-geode                             mesa
apache-opennlp                           meson
apachetop                                metaproxy
appium                                   mg
aptly                                    micronaut
arangodb                                 micropython
arduino-cli                              midnight-commander
artifactory                              mill
asciinema                                miller
ask-cli                                  mingw-w64
assimp                                   minimal-racket
astrometry-net                           minio
atdtool                                  minio-mc
atlassian-cli                            mitmproxy
aws-cdk                                  mk-configure
aws-elasticbeanstalk                     mkl-dnn
aws-sdk-cpp                              mmark
aws-shell                                molecule
awscli                                   mongo-c-driver
azure-cli                                mono
babel                                    mpc
balena-cli                               mpd
ballerina                                mpop
basex                                    mrboom
bcftools                                 mruby
bindfs                                   msmtp
bit                                      multimarkdown
bitrise                                  multitail
blast                                    mupdf
blis                                     mupdf-tools
botan                                    mutt
broot                                    mycli
byteman                                  mydumper
caffe                                    mypy
cake                                     mysql
calicoctl                                mysql@5.6
ccache                                   mysql@5.7
cdk                                      nano
cereal                                   nco
certbot                                  ncompress
certigo                                  ncview
cfn-lint                                 ndpi
chakra                                   neo4j
choose                                   netcdf
chronograf                               nethack
cimg                                     netlify-cli
citus                                    ngspice
click                                    nift
clojure                                  nnn
codemod                                  node
coffeescript                             node-build
cointop                                  node@10
collectd                                 node@12
composer                                 nodeenv
conan                                    now-cli
conjure-up                               nss
consul-template                          ntl
contentful-cli                           ntopng
convox                                   nuget
cpl                                      numpy
cppad                                    nushell
cppunit                                  oclgrind
cromwell                                 ocrmypdf
csvq                                     offlineimap
ctl                                      okteto
ctop                                     onefetch
curl                                     ooniprobe
curl-openssl                             opa
deno                                     opencv
dependency-check                         opencv@2
devspace                                 opencv@3
dfmt                                     openexr
dhall                                    openimageio
dhall-json                               openjdk
diff-pdf                                 opensc
distcc                                   openvdb
django-completion                        operator-sdk
dmd                                      osc
docfx                                    pagmo
docker-compose                           paket
docker-credential-helper-ecr             pandoc
docker-squash                            pandoc-crossref
doctl                                    passwdqc
doitlive                                 pcb
dovecot                                  pcl
doxygen                                  pcsc-lite
dub                                      pdal
duck                                     pdftoipe
dune                                     pdsh
duplicity                                peco
dvc                                      percol
easyengine                               perl
ejdb                                     perltidy
elvish                                   pgcli
embree                                   pgweb
entr                                     phpmd
envconsul                                phpmyadmin
erlang                                   phpstan
exploitdb                                phpunit
eye-d3                                   picard-tools
fades                                    picat
fail2ban                                 pipenv
fastlane                                 pipx
fastmod                                  plank
faudio                                   plantuml
fdroidserver                             pnetcdf
feedgnuplot                              pnpm
ffsend                                   pony-stable
field3d                                  poppler
firebase-cli                             pre-commit
flow                                     prestodb
fluent-bit                               primesieve
fluxctl                                  procs
flyway                                   proguard
fn                                       proj
folly                                    prometheus
fonttools                                proteinortho
fpp                                      pspg
freetds                                  pstoedit
frpc                                     puzzles
frps                                     pwntools
frugal                                   pyenv
fselect                                  pyinstaller
fuse-zip                                 pyinvoke
fx                                       pypy
gatsby-cli                               pypy3
gcc                                      python-yq
gcc@6                                    q
gcc@7                                    qmmp
gcc@8                                    quickjs
gcsfuse                                  range-v3
gdal                                     ranger
gdb                                      rav1e
ghq                                      rawtoaces
gimme-aws-creds                          re2
git-annex                                remind
git-lfs                                  rex
git-plus                                 rhino
git-quick-stats                          rke
git-remote-hg                            rocksdb
git-review                               roswell
gitbucket                                rsyslog
gitleaks                                 rtags
gitmoji                                  rtv
gjs                                      ruby-build
glassfish                                run
glfw                                     rustup-init
glib-networking                          s-nail
global                                   s3cmd
glooctl                                  saxon
glslang                                  sbt
gmic                                     scipy
gmsh                                     securefs
gmt                                      serverless
gmt@5                                    sflowtool
gnu-sed                                  shadowsocks-libev
gnunet                                   shfmt
gnuradio                                 signal-cli
go                                       sile
go@1.12                                  silk
godep                                    singular
golang-migrate                           siril
google-authenticator-libpam              sk
goreleaser                               skaffold
gr-osmosdr                               smartmontools
gradle                                   snapcraft
grafana                                  socat
grep                                     sonobuoy
grin                                     sourcekitten
grin-wallet                              spdlog
groonga                                  sphinx-doc
handbrake                                spin
helmfile                                 sqlmap
helmsman                                 starship
help2man                                 statik
hlint                                    stormssh
homebank                                 streamlink
howdoi                                   stress-ng
htslib                                   subversion
http-server                              svtplay-dl
httpie                                   swagger-codegen
hugo                                     swagger-codegen@2
icecast                                  swiftformat
ilmbase                                  swiftlint
imagemagick                              syncthing
imagemagick@6                            synfig
imgproxy                                 talloc
interactive-rebase-tool                  tarantool
internetarchive                          taskell
ipopt                                    telegraf
isl                                      teleport
istioctl                                 terraform
jdupes                                   terragrunt
jenkins                                  terrahub
jetty                                    texlab
jfrog-cli-go                             texmath
jpeg-turbo                               tintin
juju-wait                                tmuxinator
just                                     tmuxinator-completion
k3d                                      toast
kakoune                                  tomee-jax-rs
kapacitor                                tomee-plume
kawa                                     topgrade
kcov                                     tox
kepubify                                 trader
kind                                     traefik
kubeprod                                 triton
kubernetes-cli                           tunnel
kustomize                                twoping
kyma-cli                                 twtxt
languagetool                             txr
lazygit                                  typescript
ldc                                      uhd
lego                                     ultralist
leptonica                                urh
lerna                                    verilator
lftp                                     vim
libarchive                               vips
libde265                                 visp
libedit                                  vnstat
libetonyek                               volatility
libfabric                                vtk
libimagequant                            vulkan-headers
liblouis                                 wal2json
libmpdclient                             watch
libngspice                               wcslib
libofx                                   webp
libopenmpt                               weechat
libphonenumber                           whistle
librealsense                             whois
libsixel                                 wiggle
libtensorflow                            wireguard-tools
libtorch                                 wireshark
libtorrent-rasterbar                     wla-dx
libuv                                    wtf
libvirt                                  wxmaxima
libvirt-glib                             xdotool
libvoikko                                xmrig
libvterm                                 xsimd
libwebsockets                            xtensor
libxkbcommon                             yamllint
libxlsxwriter                            ykman
liquibase                                yle-dl
log4cplus                                you-get
logtalk                                  youtube-dl
lxc                                      zsh-autosuggestions
==> Deleted Formulae
apel                ipython@5           pygtk               trr
auto-scaling        leafnode            pygtkglext          uberftp
aws-elasticache     libagar             pygtksourceview     vte
aws-sns-cli         mysql-utilities     qriollo             zim
diffuse             node@8              redo
globus-toolkit      np2                 srclib
hg-flow             pygobject           terminator

Warning: You are using macOS 10.12.
We (and Apple) do not provide support for this old version.
You will encounter build failures with some formulae.
Please create pull requests instead of asking for help on Homebrew's GitHub,
Discourse, Twitter or IRC. You are responsible for resolving any issues you
experience while you are running this old version.

Error: python 3.6.5 is already installed
To upgrade to 3.7.6_1, run `brew upgrade python`.
MiniMCHD:sickbeard_mp4_automator MiniMCHD$ python manual.py --auto
Manual processor started.
/usr/local/Cellar/python@2/2.7.17_1/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python
64bit Python.
Enter path to file:
#: /Volumes/TeraPlex/AUTOMATE/DONE
Processing file /Volumes/TeraPlex/AUTOMATE/DONE/1917.2019.DVDScr.XVID.AC3.HQ.Hive-CM8[oscars]/1917.2019.DVDScr.XVID.AC3.HQ.Hive-CM8.avi
Reading video stream.
An unexpected error occurred, processing of this file has failed
'NoneType' object has no attribute 'video'
Processing file /Volumes/TeraPlex/AUTOMATE/DONE/Color.Out.of.Space.2019.1080p.SCREENER.x264-TOPKEK/Color.Out.of.Space.2019.1080p.SCREENER.x264-TOPKEK.mp4
Matched movie title as: Color Out of Space 2019-09-20
Processing Color Out of Space
An unexpected error occurred, processing of this file has failed
'NoneType' object has no attribute 'video'
Processing file /Volumes/TeraPlex/AUTOMATE/DONE/Dark.Waters.2019.1080p.SCREENER.x264.DTS-NOGRP/Dark.Waters.2019.1080p.SCREENER.x264.DTS-NOGRP.mkv
Matched movie title as: Dark Waters 2019-11-22
Processing Dark Waters
Reading video stream.
An unexpected error occurred, processing of this file has failed
'NoneType' object has no attribute 'video'
MiniMCHD:sickbeard_mp4_automator MiniMCHD$ brew upgrade python
Warning: You are using macOS 10.12.
We (and Apple) do not provide support for this old version.
You will encounter build failures with some formulae.
Please create pull requests instead of asking for help on Homebrew's GitHub,
Discourse, Twitter or IRC. You are responsible for resolving any issues you
experience while you are running this old version.

==> Upgrading 1 outdated package:
python 3.6.5 -> 3.7.6_1
==> Upgrading python 
==> Installing dependencies for python: xz
==> Installing python dependency: xz
==> Downloading https://homebrew.bintray.com/bottles/xz-5.2.4.sierra.bottle.tar.
==> Downloading from https://akamai.bintray.com/bc/bcc71ee69e2c43bf56b9c9ece5a53
######################################################################## 100.0%
==> Pouring xz-5.2.4.sierra.bottle.tar.gz
🍺  /usr/local/Cellar/xz/5.2.4: 92 files, 1MB
==> Installing python
==> Downloading https://www.python.org/ftp/python/3.7.6/Python-3.7.6.tar.xz
######################################################################## 100.0%
==> ./configure --prefix=/usr/local/Cellar/python/3.7.6_1 --enable-ipv6 --dataro
==> make
==> make install PYTHONAPPSDIR=/usr/local/Cellar/python/3.7.6_1
==> make frameworkinstallextras PYTHONAPPSDIR=/usr/local/Cellar/python/3.7.6_1/s
==> Downloading https://files.pythonhosted.org/packages/f7/b6/5b98441b6749ea1db1
######################################################################## 100.0%
==> Downloading https://files.pythonhosted.org/packages/ce/ea/9b445176a65ae4ba22
Already downloaded: /Users/MiniMCHD/Library/Caches/Homebrew/downloads/0b4dcf92e1664a17fefc4c0f7993a9943849d88144a7a22c4b5458271c1527ea--pip-19.3.1.tar.gz
==> Downloading https://files.pythonhosted.org/packages/59/b0/11710a598e1e148fb7
Already downloaded: /Users/MiniMCHD/Library/Caches/Homebrew/downloads/98eb50a86909635dacc94e7f91048dd6ab4bc084cf23523a6ff780e11e0d3ec1--wheel-0.33.6.tar.gz
==> /usr/local/Cellar/python/3.7.6_1/bin/python3 -s setup.py --no-user-cfg insta
==> /usr/local/Cellar/python/3.7.6_1/bin/python3 -s setup.py --no-user-cfg insta
==> /usr/local/Cellar/python/3.7.6_1/bin/python3 -s setup.py --no-user-cfg insta
==> Caveats
Python has been installed as
  /usr/local/bin/python3

Unversioned symlinks `python`, `python-config`, `pip` etc. pointing to
`python3`, `python3-config`, `pip3` etc., respectively, have been installed into
  /usr/local/opt/python/libexec/bin

If you need Homebrew's Python 2.7 run
  brew install python@2

You can install Python packages with
  pip3 install <package>
They will install into the site-package directory
  /usr/local/lib/python3.7/site-packages

See: https://docs.brew.sh/Homebrew-and-Python
==> Summary
🍺  /usr/local/Cellar/python/3.7.6_1: 8,586 files, 120.4MB, built in 4 minutes 40 seconds
Removing: /usr/local/Cellar/python/3.6.5... (4,765 files, 100.1MB)
==> Checking for dependents of upgraded formulae...
+=> No dependents found!
==> Caveats
==> python
Python has been installed as
  /usr/local/bin/python3

Unversioned symlinks `python`, `python-config`, `pip` etc. pointing to
`python3`, `python3-config`, `pip3` etc., respectively, have been installed into
  /usr/local/opt/python/libexec/bin

If you need Homebrew's Python 2.7 run
  brew install python@2

You can install Python packages with
  pip3 install <package>
They will install into the site-package directory
  /usr/local/lib/python3.7/site-packages

See: https://docs.brew.sh/Homebrew-and-Python
MiniMCHD:sickbeard_mp4_automator MiniMCHD$

**Comment by evanwins** [comment]
Okay, I was able to get it working again by replacing the ffmpeg and ffprobe binaries. Not sure what the problem was but I upgraded them from 3.4.1 to 4.2.1 so there's that. 

This script is a massive part of my set up. It makes every file played by Plex playback smoothly, whether local or remote. By limiting the amount of transcoding Plex has to do on the fly it severely changes how the content is played back and I'm very grateful for it. 

mdhiggins, if you see this I'd like to donate to you so let me know how I can do that.
